### PR TITLE
add here2 mag to startup for fmu-v5

### PR DESCRIPTION
**Pull request by sfuhrer**
The here2 mag currently gets only started on fmu-v3 (as this is the fmu version that gets sold with the here2. Would be nice to also enable using it on a v5 from my point of view. 

Note: one needs to change the connector (from 8 pin to 10 pin) to get it connectable to a Pixhawk4.

With this PR it also starts up on fmu-v5. I've simply copied it from fmu-v3 (https://github.com/PX4/Firmware/blob/803a71928f106ccefb453fc2bdf960fb22a63a89/boards/px4/fmu-v3/init/rc.board_sensors#L69-L74). 

@dagar what are your thoughts, is there something against starting it also for v5?

### CODE REVIEW

File: boards/px4/fmu-v5/init/rc.board_sensors [hunk 1]
```diff
@@ -22,6 +22,13 @@ hmc5883 -C -T -X start
 qmc5883 -X start
 lis3mdl -X start
 
+# ICM20948 as external magnetometer on I2C (e.g. Here GPS)
+if ! icm20948 -X -M -R 6 start
+then
+  # external emulated AK09916 (Here2) is rotated 270 degrees yaw
+  ak09916 -X -R 6 start
```
Comment (dagar) [inline boards/px4/fmu-v5/init/rc.board_sensors]: Is the AK09916 rotated 270 degrees in your particular setup or within the actual Here2?

**Comment by dagar** [comment]
> @dagar what are your thoughts, is there something against starting it also for v5?

No, although I think it might be time for a more unified system to probe the possible external peripherals. The other option is adding parameters to explicitly enable them.

I'm just curious, have you tried UAVCAN on this thing instead of I2C?

File: boards/px4/fmu-v5/init/rc.board_sensors (hunk 1 above)
Comment (sfuhrer) [reply boards/px4/fmu-v5/init/rc.board_sensors]: Seems like inside the Here2, I just copied all this from the fmu-v3 init

**Comment by sfuhrer** [comment]
> > @dagar what are your thoughts, is there something against starting it also for v5?
> 
> No, although I think it might be time for a more unified system to probe the possible external peripherals. The other option is adding parameters to explicitly enable them.
> 
> I'm just curious, have you tried UAVCAN on this thing instead of I2C?



> > @dagar what are your thoughts, is there something against starting it also for v5?
> 
> No, although I think it might be time for a more unified system to probe the possible external peripherals. The other option is adding parameters to explicitly enable them.
> 
> I'm just curious, have you tried UAVCAN on this thing instead of I2C?

No, not yet. What would again be the advantages in this case against I2C?

File: boards/px4/fmu-v5/init/rc.board_sensors (hunk 1 above)
Comment (dagar) [reply boards/px4/fmu-v5/init/rc.board_sensors]: That's kind of horrible depending on if that's specific to the Here2 or the driver implementation. We'll likely have to review this later (especially if someone else starts using an AK09916).

**Comment by dagar** [comment]
> No, not yet. What would again be the advantages in this case against I2C?

It's more robust and has the potential for updating firmware over CAN as well. My understanding of the Here2 is that this is a fake "AK09916" being presented by the processor in the unit.

**Pull request merged**